fs/operations/check.go

```diff
@@ -38,6 +38,7 @@ type CheckOpt struct {
 	Fdst, Fsrc   fs.Fs     // fses to check
 	Check        checkFn   // function to use for checking
 	OneWay       bool      // one way only?
+	ReportDirs   bool      // report dir differences in addition to files
```

Comment:
This changes the `rc` interface, so the docs need fixing up here

https://github.com/rclone/rclone/blob/94997d25d2d8fa37a3615af5598bd9fb50e19bd5/fs/operations/rc.go#L721